sapp/ui/tests/interactive_test.py

```diff
@@ -1786,6 +1787,7 @@ def testCreateIssueOutputStringNoSourcesNoSinks(self):
             source_kinds=None,
             sink_names=None,
             sink_kinds=["sink1", "sink2"],
+            status=IssueStatus.UNCATEGORIZED,
```

Comment:
Can we add a test similarly to what we did for [testListIssuesFilterCodes]( https://github.com/facebook/sapp/blob/e30857392d6ab2f8e399274d41b7a957678ea511/sapp/ui/tests/interactive_test.py#L205)

Reply:
I will write one and get back :)

Reply:
Done :)